validate/josev2/validator.go

```diff
@@ -14,21 +14,11 @@ import (
 type Validator struct {
 	keyFunc            func(context.Context) (interface{}, error) // Required.
 	signatureAlgorithm jose.SignatureAlgorithm                    // Required.
-	expectedClaims     func() jwt.Expected                        // Optional.
-	customClaims       func() CustomClaims                        // Optional.
+	expectedClaims     jwt.Expected                               // Optional.
```

Comment:
Same as above, can we do this without requiring explicitly the `jwt.Expected` type?